pkg/server/cors.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+import (
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/grpc-ecosystem/grpc-gateway/utilities"
+)
+
+const (
+	// At time of writing, the version argument is required to be 1.
```

Comment:
can you explain why or link to something that does?

Reply:
added.